docs/sources/installation/rpm.md

```diff
@@ -23,6 +23,8 @@ Read [Upgrading Grafana]({{< relref "installation/upgrading.md" >}}) for tips an
 
 ## Install Stable
 
+See [https://grafana.com/grafana/download?platform=linux](https://grafana.com/grafana/download?platform=linux) for the current download and installation instructions.
```

Comment:
we have this link in the download table above this paragraph already, so not sure we need this here?